src/layouts/Default.vue

```diff
@@ -5,6 +5,7 @@
       <v-spacer />
       <div>{{ webOcTitle }}</div>
       <v-spacer />
+      <DisclaimerPopup />
```

Comment:
We probably need to fix the Z-index to be in front of at least the search locations button and the sidebar

Reply:
This is done in the SplashScreen pull request